workflow/automation/lib/schedulers/slurm.py

```diff
@@ -22,7 +22,7 @@ def check_queues(self, user: bool = False, target_machine: HPC = None):
         )
 
         if target_machine is None:
-            target_machine = self.current_machine
+            target_machine = HPC[self.current_machine]  # self.current_machine is str
```

Comment:
self.current_machine is a string, whereas target_machine should be of HPC enum type to enable line 27
```
account = self.account[target_machine.name]
```